elfi/methods/inference/samplers.py

```diff
@@ -347,7 +347,7 @@ def __init__(self, model, discrepancy_name=None, output_names=None, **kwargs):
         self._round_random_state = None
 
     def set_objective(self, n_samples, thresholds=None, max_iter=10,
-                      adaptive_threshold=False, initial_quantile=0.20,
+                      adaptive_quantile=False, initial_quantile=0.20,
```

Comment:
I'd prefer `adaptive_threshold` over `adaptive_quantile` due to analogy to our "non-adaptive" version of abc-smc (controlled with `thresholds`), but also `adaptive_iterations` would be ok. However, I do agree that `adaptive_quantile` goes better with the parameter `initial_quantile`.

Reply:
`adaptive_threshold` is still used in the code to indicate the situation where we do not have predetermined thresholds but use an adaptive threshold ie same as previous version. the reason i removed it from input variables is that it was not used: we see it as an input variable in the previous version here, but in the next lines it was overwritten based on whether or not `thresholds` is none. thus `thresholds` was the real control variable in the previous version. now the alternative fix would have been not to remove the unused variable `adaptive_threshold` but to take its value into account. is this the preferred fix?

about the new variable `adaptive_quantile`: i named it so because it controls the decision about quantile adaptation ie when we have `adaptive_threshold` but not `adaptive_quantile`, we do not have predetermined threshold values but also no quantile adaptation based on density ratio estimation. i do not think that this variable should be called `adaptive_threshold` since we can have adaptive threshold without quantile adaptation. however it is true that the variable could be called `adaptive_iterations`, since when we use an adaptive quantile, we will also stop iterations in case the adaptive quantile value exceeds the predetermined threshold vs when we have an adaptive threshold without quantile adaptation, num iterations is predetermined. and it is true that the adaptive iterations aspect could be more important to a user, so the control variable name should be based on that. should i update `adaptive_quantile` --> `adaptive_iterations`?